README.md

```diff
@@ -10,15 +10,34 @@ This is a fully batteries-included development environment for Windows on Kubern
 - Windows binaries for kube-proxy.exe and kubelet.exe that are fully built from source (K8s main branch)
 - kubeadm installation that can put the bleeding-edge Linux control plane in place, so you can test new features like privileged containers
 
-# Quick start
+# Quick Start
 
-- clone this repo (obviously!)
-- install vagrant & virtualbox (the base tools for this project)
+- Clone this repo (obviously!)
+- Install [Vagrant](https://www.vagrantup.com/downloads) & [VirtualBox](https://www.virtualbox.org/wiki/Downloads) (the base tools for this project)
 - `vagrant plugin install vagrant-reload winrm winrm-elevated`, vagrant-reload needed to easily reboot windows VMs during setup of containers features.
 - `make all`, this will create the entire cluster for you and compile windows binaries from source
-- if the above failed, run `vagrant provision winw1`, just in case you have a flake during windows installation.
+- If the above failed, run `vagrant provision winw1`, just in case you have a flake during windows installation.
 - `vagrant ssh controlplane` and run `kubectl get nodes` to see your running dual-os linux+windows k8s cluster.
 
+## Windows
```

Comment:
@luthermonson can you state this is running on WSL, and not plain Windows.

Reply:
ya no problemo